### PR TITLE
updates to content: unsigned apps in mac

### LINKED ISSUE #28: More instructions for macs
> Hi Anthony,
> 
> Unfortunately, I’m stuck at the trailing edge of technology on High Sierra so that option was not available in my system settings.
> 
> However, I discovered that I could unlock the gatekeeper with the following line in the Terminal:
> 
> sudo spctl --master-disable
> 
>  
> 
> So I have now managed to install everything.

Can we check if this correct and if so add a suitable note to our mac terminal document for High Sierra.

### PR DESCRIPTION
**Pull request by andrew-1234**
- adds details for older macOS versions / current versions
- Fixes #28

### CODE REVIEW

File: content/resources/help-centre/software/unsigned/index.md [hunk 1]
```diff
@@ -23,6 +27,30 @@ unlock button, and then tick Terminal:
...
+wish to enable the protection again later, you can use:
+
+```sh
+> sudo spctl --master-enable
+```
+
+Note that once you have successfully opened and run an unsigned program after
+disabling the protection, that program will continue to be available, even after
+you re-enable the protection. 
+
```
Comment (atruskie) [inline content/resources/help-centre/software/unsigned/index.md:53]: Ok I researched this. This is a more dangerous change: it disabled gatekeeper system wide and not just for terminal apps.

We should add a strong warning (`danger`) that this is not an ideal solution, and `spctl` should be reenabled as soon as we're done allowing the target files to run.

**Review by atruskie — CHANGES_REQUESTED** [review]
While researching I found this page:  https://disable-gatekeeper.github.io/

It has a lot of good information on it, perhaps we should link to it.

Nit pick: your commit message is malformed, it should read `Fixes #28` and not just `#28`. When you do that the associated issue is automatically closed. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: content/resources/help-centre/software/unsigned/index.md (hunk 1 above)
Comment (andrew-1234) [reply content/resources/help-centre/software/unsigned/index.md:53]: Ok  i'll add a warning to this. 

That github link is super helpful. I tested their example to disable the quarantine for a single app using 
`xattr -dr com.apple.quarantine ./emu` 
And this worked for me, letting me run emu while my gatekeeper was enabled. So i'll add that in as well, and the link to that github.

File: content/resources/help-centre/software/unsigned/index.md [hunk 2]
```diff
@@ -23,6 +28,66 @@ unlock button, and then tick Terminal:
...
+```sh
+> xattr -dr com.apple.quarantine <your-path-here>
+```
+
+Replace \<your-path-here\> with the location of the executable file you would
+like to allow. For example, the following code will allow the `emu` executable
+to run, which in this example is located in `~/emu-program/`:
+
+```sh
+xattr -dr com.apple.quarantine ~/emu-program/emu
```
Comment (atruskie) [inline content/resources/help-centre/software/unsigned/index.md:53]:
```suggestion
to run, which in this example is located in `~/emu/`:

```sh
xattr -dr com.apple.quarantine ~/emu/emu
```

File: content/resources/help-centre/software/unsigned/index.md [hunk 3]
```diff
@@ -23,6 +28,66 @@ unlock button, and then tick Terminal:
...
+### Allow a single program
+
+The first method will allow a single program to run, without having to disable
+Gatekeeper. Open a terminal and run the following command:
+
+```sh
+> xattr -dr com.apple.quarantine <your-path-here>
+```
+
+Replace \<your-path-here\> with the location of the executable file you would
```
Comment (atruskie) [inline content/resources/help-centre/software/unsigned/index.md:48]:
```suggestion
Replace `<your-path-here>` with the location of the executable file you would
```

**Review by atruskie — APPROVED** [review]
great work

**Pull request merged**